helpme/templates/helpme/ticket_detail.html

```diff
@@ -141,7 +141,8 @@ <h3 class="mt-3">
             <tr>
               <td>{{ item.action }}</td>
               <td>{{ item.username }}</td>
-              <td>{{ item.time }}</td>
+              {% convert_isotime item.time as datetime %}
+              <td>{{ datetime|date:"M j, Y, g:i a" }}</td> 
```

Comment:
This works.
If you want to make it even cleaner, you could make a new filter that returns a datetime object that you can then chain into the 'date' filter.